src/include/migraphx/module.hpp

```diff
@@ -69,6 +69,8 @@ struct module
 
     std::string name() const;
 
+    std::string get_target() const;
+    void set_target(const std::string& target_name);
```

Comment:
This should be removed. Its not used, and doesnt make sense on modules that are for multiple targets.

Reply:
Those target annotations are only for the root module. If it contains further nested submodules that are for different targets, then those modules themselves would have a `target` attribute.

Reply:
But `get_target` is never used anywhere(except in the unit test), we should add it only if we need it.